templates/resources/sso/service.yaml

```diff
@@ -13,6 +13,6 @@ spec:
   - name: https
     port: 8443
     protocol: TCP
-    targetPort: 8443
+    targetPort: https
   selector:
-    name: sso
+    name: keycloak
```

Comment:
Should the openshift service name be switched to the upstream keycloak?

Reply:
Looking at `jboss-amqmaas-image/templates/resources` we have both `sso` and `standard-authservice`.   Are both required?